src/fileformat/types/import_table/elf_import_table.cpp

```diff
@@ -0,0 +1,96 @@
+/**
+ * @file include/retdec/fileformat/types/import_table/elf_import_table.h
+ * @brief Class for ELF import table.
+ * @copyright (c) 2021 Avast Software, licensed under the MIT license
+ */
+
+#include "retdec/fileformat/utils/crypto.h"
+#include "retdec/utils/string.h"
+#include "retdec/fileformat/utils/tlsh/tlsh.h"
+#include "retdec/fileformat/types/import_table/elf_import_table.h"
+#include <algorithm>
+#include <regex>
+
+using namespace retdec::utils;
+
+namespace retdec {
+namespace fileformat {
+
+static const std::set<std::string> exclusion_set = {
+	"__libc_start_main", // main function
+	"main", // main function
+	"abort", // ARM default
+	"cachectl", // MIPS default
+	"cacheflush", // MIPS default
+	"puts", // Compiler optimization (function replacement)
+	"atol", // Compiler optimization (function replacement)
+	"malloc_trim" // GNU extensions
+};
+
+bool isSymbolExcluded(std::string symbol)
```

Comment:
Why pass string as value? Are you modifying it in the function? If not then `const std::string&` or `std::string_view`.